gradle.properties

```diff
@@ -1,10 +1,9 @@
 GROUP=com.segment.analytics.android.integrations
 
-VERSION_CODE=141
-VERSION_NAME=1.4.1-SNAPSHOT
+VERSION = 1.4.1-SNAPSHOT
 
 POM_ARTIFACT_ID=localytics
-POM_PACKAGING=aar
+POM_PACKAGING=jar
```

Comment:
Was anyone else using the `aar` version of this? Should we instead be generating both an `aar` and `jar` so we don't regress for other customers. Sorry not familiar with this integration.

Reply:
Good idea!

Reply:
Looks like customers can pull in the dep as a jar by appending `@jar` in their build.gradle file, for example: `implementation 'com.segment.analytics.android.integrations:localytics:1.4.0@jar'`. So, if the root cause of some customers' issues is really solved by pulling in a `jar` rather than an `aar`, that should do the trick.